lib/bot.js

```diff
@@ -251,6 +251,11 @@ class Bot {
       return;
     }
 
+    if (message.bot_id == 'B42KVL3PF') { // CHANGEME
```

Comment:
I couldn't find a way of fetching our own bot ID either in the docs, and matching up our own messages would be fairly involved (from having a cursory glance at how the bot works) - being able to ignore a list of bot IDs would be an easy way to accomplish our goal in a way that doesn't involve hardcoding or significant refactoring.